app/src/main/java/mozilla/lockbox/action/RouteAction.kt

```diff
@@ -28,8 +27,11 @@ sealed class RouteAction(
     object LockScreen : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.lock_screen)
     object Filter : RouteAction(TelemetryEventMethod.tap, TelemetryEventObject.filter)
     data class ItemDetail(val id: String) : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.entry_detail)
-    data class OpenWebsite(val url: String) : RouteAction(TelemetryEventMethod.tap, TelemetryEventObject.open_in_browser)
-    data class SystemSetting(val setting: SettingIntent) : RouteAction(TelemetryEventMethod.show, TelemetryEventObject.settings_system)
+    data class OpenWebsite(val url: String) :
+        RouteAction(TelemetryEventMethod.tap, TelemetryEventObject.open_in_browser)
+
+    data class SystemSetting(val setting: SettingIntent) :
+        RouteAction(TelemetryEventMethod.show, TelemetryEventObject.settings_system)
```

Comment:
These line breaks were enraging the linter. Apologies for the 👾

Reply:
`./gradlew ktlintFormat` and chill.